scripts/shuffle/plot.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2023 MosaicML Streaming authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Plot results of comparing streaming dataset shuffling algorithms."""
+
+from argparse import ArgumentParser, Namespace
+from typing import Dict, List, Tuple
+
+from matplotlib import pyplot as plt
+
+mark_sizes = [
+    ('ImageNet (1.2M)', 1_281_167, '#cfc'),
+    ('C4 (365M)', 364_868_892, '#8c8'),
+    ('LAION-5B (5B)', 5_000_000_000, '#484'),
+]
+
+mark_times = [
+    ('Instant (1s)', 1, '#bdf'),
+    ('Fast (10s)', 10, '#9ce'),
+    ('Slow (60s)', 60, '#08f'),
+]
+
+algo_colors = {
+    'py1x': 'red',
+    'py2x': 'purple',
+}
+
+
+def parse_args() -> Namespace:
+    """Parse command-line arguments.
+
+    Returns:
+        Namespace: Command-line arguments.
+    """
+    args = ArgumentParser()
+    args.add_argument('--in', type=str, required=True)
```

Comment:
Please add a `help` section on what `--in` and `--out` means. Thanks!

Reply:
done